Core/Object Arts/Dolphin/IDE/Base/HelpImagesCreationTest.cls

```diff
@@ -321,20 +321,20 @@ testSystemFolder
 	(folderPresenter view)
 		lvmSetBkImage: LVBKIMAGEW new;
 		backcolor: (Color 
-					red: 255
-					green: 255
-					blue: 238).
+					red: 41
```

Comment:
Use `(Color named: #smalltalkSystem)` rather than hardcoding. I know it was done that way before, but by using the symbolic colour it won't need to be updated again if this is changed.